include/exadg/acoustic_conservation_equations/postprocessor/sound_energy_calculator.h

```diff
@@ -200,9 +200,9 @@ class SoundEnergyCalculator
     CellIntegratorP pressure(matrix_free_in, dof_index_pressure, quad_index);
     CellIntegratorU velocity(matrix_free_in, dof_index_velocity, quad_index);
 
-    Number const rho       = static_cast<Number>(data.density);
+    Number const rho_inv   = static_cast<Number>(1.0 / data.density);
     Number const c         = static_cast<Number>(data.speed_of_sound);
-    Number const rhocc_inv = Number{1.0} / (rho * c * c);
+    Number const rhocc_inv = rho_inv / (c * c);
```

Comment:
I could not find `c` in the code below, so maybe this is better:

```C++
Number const rho_inv   = static_cast<Number>(1.0 / data.density);
Number const rhocc_inv = static_cast<Number>(1.0 / (data.density*data.speed_of_sound*data.speed_of_sound);
```

The variable name `rhocc_inv` would then also be clearer to me.

Reply:
[caabe6f](https://github.com/exadg/exadg/pull/631/commits/caabe6f046a1795b4e88879d583d111aff8de496)